### PR TITLE
chore: add repo metadata file

### PR DESCRIPTION
**Pull request by anguillanneuf**

### CODE REVIEW

**Review by samarthsingal — APPROVED** [review]

**Comment by anguillanneuf** [comment]
Thanks @samarthsingal. I will wait for a second approval from the @googleapis/yoshi-java folks because this file is a little picky.

**Review by meltsufin — APPROVED** [review]

**Pull request merged**
